bin/cron/workshops_to_s3

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+require_relative '../../dashboard/config/environment'
+require_relative '../../lib/cdo/only_one'
+require 'cdo/aws/s3'
+
+# Automates the export of 5-day summer workshops data
+# to an S3 bucket. Mirrors what would happen via the UI
+# if an admin clicked the "Download CSV" option
+# in the workshop dashboard.
+def main
+  workshops = Pd::Workshop.
+    where(
+      {
+        course: [Pd::Workshop::COURSE_CSP, Pd::Workshop::COURSE_CSD],
+        subject: Pd::Workshop::SUBJECT_SUMMER_WORKSHOP
+      }
+    ).
+    scheduled_start_on_or_after(Date.new(2019, 5, 1))
```

Comment:
Not super important, but I think I would have styled this like this based on our conventions:
```workshops = Pd::Workshop.
    where({
        course: [Pd::Workshop::COURSE_CSP, Pd::Workshop::COURSE_CSD],
        subject: Pd::Workshop::SUBJECT_SUMMER_WORKSHOP
      }).scheduled_start_on_or_after(Date.new(2019, 5, 1))
```